portfolio/src/main/webapp/script.js

```diff
@@ -105,6 +105,16 @@ function getComments() {  // eslint-disable-line no-unused-vars
       });
 }
 
+/**
+ * Delete all the comments stored server-side and update the home page
+ * to reflect these changes.
+ */
+function deleteComments() {  // eslint-disable-line no-unused-vars
+  fetch('/delete-comments', {method: 'POST'}).then(() => {
+    getComments();
+  });
```

Comment:
() => void getComments();

Read about arrow functions in https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions. Pay attention to the parts about the void operator and the single expression syntax.

Reply:
Okay